pages/blog.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>A yr of Code Blog | Read me </title>
+      <link rel="stylesheet" href="styles/style.css">
+      <link rel="stylesheet" media="screen and (max-width: 768px)" href="css/mobile.css">

```

Comment:
Cool way to pull in your mobile-specific styles, but your path is invalid (should be `styles/mobile.css`)